### PR TITLE
feat(crank): drop duplicated build/push commands

### PR DESCRIPTION
**Pull request by phisco**
### Description of your changes

Address the uncontroversial part of https://github.com/crossplane/crossplane/issues/4783, dropping `build` and `push` subcommands in favor of the implementations at `xpkg build` and `xpkg push`. Left moving install and update to a later stage when a decision on https://github.com/crossplane/crossplane/issues/4783 has been taken, just to avoid moving them around twice.

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Added or updated unit **and** E2E tests for my change.
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] ~Added `backport release-x.y` labels to auto-backport this PR, if necessary.~
- [ ] ~Opened a PR updating the [docs], if necessary.~

[contribution process]: https://git.io/fj2m9
[docs]: https://docs.crossplane.io/contribute/contribute

### CODE REVIEW

File: cmd/crank/install.go [hunk 1]
```diff
@@ -69,11 +69,6 @@ type installCmd struct {
 	Provider      installProviderCmd `cmd:"" help:"Install a Provider package."`
 }
 
-// Run runs the install cmd.
-func (c *installCmd) Run(_ *buildChild) error {
-	return nil
-}
```
Comment (negz) [inline cmd/crank/install.go:75]: Was this intentional? I see you said you were leaving install for now - I'm guessing this is a no-op change?

File: cmd/crank/update.go [hunk 2]
```diff
@@ -28,11 +28,6 @@ type updateCmd struct {
 	Provider      updateProviderCmd `cmd:"" help:"Update a Provider package."`
 }
 
-// Run runs the update cmd.
-func (c *updateCmd) Run(_ *buildChild) error {
-	return nil
-}
```
Comment (negz) [inline cmd/crank/update.go:34]: Same question here.

**Review by negz — APPROVED** [review]
Thanks @phisco!

File: cmd/crank/install.go (hunk 1 above)
Comment (phisco) [reply cmd/crank/install.go:75]: Yes, the actual commands are implemented in the subcommands' Run. Not really sure why these were there.

**Pull request merged**